designsystem.digdir.no/components/SidebarMenu/SidebarMenu.tsx

```diff
@@ -1,67 +1,45 @@
 import React from 'react';
+import Link from 'next/link';
+import cn from 'classnames';
 
 import { Tag } from '../Tag/Tag';
+import { capitalizeString } from '../../utils/StringHelpers';
 
 import classes from './SidebarMenu.module.css';
 
 interface SidebarMenuProps {
   title: string;
+  items: any;
```

Comment:
Could we avoid using any?😄 I think is better to create a type for the menu instead of using any here.